### PR TITLE
Added style configurations for most of inner heatmap html elements

### PR DESCRIPTION
**Pull request by andreysaleba**

### CODE REVIEW

File: src/index.jsx [hunk 1]
```diff
@@ -259,7 +259,7 @@ class CalendarHeatmap extends React.Component {
 
   renderWeek(weekIndex) {
     return (
-      <g key={weekIndex} transform={this.getTransformForWeek(weekIndex)}>
+      <g key={weekIndex} transform={this.getTransformForWeek(weekIndex)} className='week'>
```
Comment (okize) [inline src/index.jsx]: could we make this className a little more verbose/specific?

**Comment by okize** [comment]
@andreysaleba I like this! My only concern is the possibility of class name collisions (`week`, for example, seems like a class that could easily exist in a consuming application). I wonder if we could prepend the classes with `calendar-heatmap` or something? ie. a pseudo-namespace to avoid unwanted inheritance.

**Comment by andreysaleba** [comment]
sure, I'm not so good with dat css things, I'l fix it in a while

**Comment by andreysaleba** [comment]
@okize, I've added prefix for classnames.

**Comment by andreysaleba** [comment]
Oh, I've forgot something

**Comment by andreysaleba** [comment]
I've added the namespace now @okize

**Comment by okize** [comment]
looks great, thanks @andreysaleba!

**Comment by okize** [comment]
published as v1.6.0

**Pull request merged**